### ISSUE TITLE
styling fix for menu styles 

### ISSUE THREAD

**Issue by likello** [osu-wams/madrone#231]
when you place a menu in the primary menu region, it doesn't receive the proper styling and doesn't provide a very noticeable hover state.

here is an example: https://drupal.dev.oregonstate.edu/

here are the styles that I have used in the homepage that matches the behavior or the hover state for a regular main menu or group menu: 
```

.nav-link:hover {
	background-color: #fff;
	color: $neutral-700;
}

.nav-link:active {
	background-color: $paddletailblack;
  color: #fff;
}

.nav-link:focus {
	color: $paddletailblack;
}
```
